src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements BehaviorFigure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: "
+                + "circle, "
+                + "area: "
+                + getArea()
+                + " sq.units, "
+                + "radius: "
+                + radius
+                + " units, "
+                + "color: "
+                + getColor();
```

Comment:
```suggestion
    public void draw() {
        System.out.println("Figure: "
                + "circle, "
                + "area: "
                + getArea()
                + " sq.units, "
                + "radius: "
                + radius
                + " units, "
                + "color: "
                + getColor());
```
let's create a separate interface Drawable with method draw()
and make Figure class implement 2 interfaces, because the purposes of the interfaces are different: 1st calculates the area, second is printing out whole information